src/sections/Footer/index.js

```diff
@@ -1,170 +1,219 @@
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    GSOC 2020
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    GSOD 2020
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Community Bridge
+                  </Link>
+                </li>
+              </ul>
+            </div>
+          </Col>
+          <Col xs={6} sm={4} lg={3}>
+            <div className="footer-widgets">
+              <h3 className="widget-title">Company</h3>
+              <ul className="widget-catagory">
+                <li>
+                  <Link className="anchor" to="#">
+                    News
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Brand
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Careers
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Partners
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Contact
+                  </Link>
+                </li>
+              </ul>
+            </div>
+          </Col>
+          </Row>
+          <Row className="footer-contact">
+          <Col className="footer-widgets" xs={12} sm={6} lg={7}>
+            <Language />
+          </Col>
+          <Col className="footer-widgets" xs={12} sm={6} lg={5}>
+            <h3 class="widget-title">Contact</h3>
+            <Row className="contactf">
+              <Link className="social-icons" to="#">
+                <img height="40 px" src={mail_icon} />
+              </Link>
+              <Link className="social-icons" to="#">
+                <img height="40 px" src={slack_icon} />
+              </Link>
+              <Link className="social-icons" to="#">
+                <img height="40 px" src={twitter_icon} />
+              </Link>
+              <Link className="social-icons" to="#">
+                <img height="40 px" src={github_icon} />
+              </Link>
+              <Link className="social-icons" to="#">
+                <img height="40 px" src={youtube_icon} />
+              </Link>
+              <Link className="social-icons" to="#">
+                <img height="40 px" src={docker_icon} />
+              </Link>
+              </Row>
+          </Col>
+        </Row>
+          </Col>
+        </Row>
+        <div className="footer-bottom">
+          <ul className="footer-menu">
+            <li>
+              <Link className="anchor" to="#">
+                Privacy

```

Comment:
Can they be populated dynamically using sitemap?

Reply:
Do you mean the sitemap plugin for gatsby?

Reply:
Yes I guess, data should be populated from the sitemap rather than hardcoding. But if that is not feasible right now we can move ahead with the PR as a first iteration.

Reply:
Good suggestion about the sitemap. It'll be too dynamic for the footer, though, considering how sprawling the site will be. The footer will need manual curation.